app/controllers/api/v1/cookie_and_jwt_api/person_controller.rb

```diff
@@ -10,6 +10,8 @@ def me
           render json: current_user, serializer: Api::PersonSerializer
         end
 
+        delegate :my_students, to: :current_user
```

Comment:
What does this do exactly? Does this render current_user.my_students as a json? Without using the serializer for it? Is that what we want?